modules/virt-importing-vm-datavolume.adoc

```diff
@@ -4,82 +4,82 @@
+** A virtual machine disk image in RAW, ISO, or QCOW2 format, optionally compressed by using `xz` or `gz`.
+** An HTTP or HTTPS endpoint where the image is hosted, along with any authentication credentials needed to access the data source.
+* To import a container disk, you must have a virtual machine image built into a container disk and stored in a container registry, along with any authentication credentials needed to access the data source.
+* If the virtual machine must communicate with servers that use self-signed certificates or certificates not signed by the system CA bundle, you must create a config map in the same namespace as the data volume.
 
 .Procedure
 
-. Optional: If your data source requires authentication credentials, edit the
-`endpoint-secret.yaml` file, and apply the updated configuration to the cluster:
+. If your data source requires authentication, add the credentials to your cluster:
+
+.. Create a `Secret` manifest, specifying the data source credentials:
 +
 [source,yaml]
 ----
 apiVersion: v1
 kind: Secret
 metadata:
-  name: <endpoint-secret>
+  name: <endpoint-secret> <1>
   labels:
     app: containerized-data-importer
 type: Opaque
 data:
-  accessKeyId: "" <1>
-  secretKey:   "" <2>
+  accessKeyId: "" <2>
+  secretKey:   "" <3>
 ----
-<1> Optional: your key or user name, base64 encoded
-<2> Optional: your secret or password, base64 encoded
+<1> Specify the name of the `Secret`.
+<2> Specify the Base64-encoded key ID or user name.
+<3> Specify the Base64-encoded secret key or password.
+
+.. Apply the `Secret` manifest:
 +
 [source,terminal]
 ----
-$ oc apply -f endpoint-secret.yaml
+$ oc apply -f <endpoint-secret>.yaml
 ----
 
-. Edit the virtual machine configuration file, specifying the data source for
-the virtual machine image you want to import. In this example, a Fedora image is imported from an `http` source:
+. Edit the `VirtualMachine` manifest, specifying the data source for
+the virtual machine image you want to import:
 +
+[%collapsible]
 [source,yaml]
+====
 ----
 apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
   creationTimestamp: null
   labels:
     kubevirt.io/vm: vm-fedora-datavolume
-  name: vm-fedora-datavolume
+  name: <vm_fedora_datavolume> <1>
 spec:
   dataVolumeTemplates:
   - metadata:
       creationTimestamp: null
-      name: fedora-dv
+      name: <fedora_dv> <2>
     spec:
-      pvc:
-        accessModes:
-        - ReadWriteOnce
+      storage:
         resources:
           requests:
             storage: 10Gi
         storageClassName: local
       source:
-        http: <1>
-          url: "https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2" <2>
-          secretRef: "" <3>
-          certConfigMap: "" <4>
+        http: <3>
+          url: <"https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"> <4>
```

Comment:
fedora 33 is really old, maybe point to something more recent like this https://mirror.arizona.edu/fedora/linux/releases/35/Cloud/x86_64/images/Fedora-Cloud-Base-35-1.2.x86_64.qcow2